src/satosa/micro_services/account_linking.py

```diff
@@ -94,15 +96,17 @@ def process(self, context, internal_response):
         # Store the issuer subject_id/sub because we'll need it in handle_al_response
         internal_response.attributes['issuer_user_id'] = internal_response.subject_id
         if status_code == 200:
-            satosa_logging(logger, logging.INFO, "issuer/id pair is linked in AL service",
-                           context.state)
+            msg = "issuer/id pair is linked in AL service"
+            logline = lu.LOG_FMT.format(id=lu.get_session_id(context.state), message=msg)
+            logger.info(logline)
             internal_response.subject_id = message
             data['user_id'] = message
             if self.id_to_attr:
                 internal_response.attributes[self.id_to_attr] = [message]
         else:
-            satosa_logging(logger, logging.INFO, "issuer/id pair is not linked in AL service. Got a ticket",
-                           context.state)
+            msg = "issuer/id pair is not linked in AL service. Got a ticket"
+            logline = lu.LOG_FMT.format(lu.get_session_id(context.state), message=msg)
```

Comment:
again for the sake of obviousness (since you've probably seen this from the failing tests) this needs to be written as `id=lu.get_session_id(context.state), message=msg`